param/parameterized.py

```diff
@@ -3593,169 +3711,8 @@ def __str__(self):
         """Return a short representation of the name and class of this object."""
         return f"<{self.__class__.__name__} {self.name}>"
 
-    # PARAM3_DEPRECATION
-    @_deprecated(extra_msg="Use instead `.param.pprint()`")
-    def script_repr(self,imports=[],prefix="    "):
```

Comment:
@maximlt Now the methods are removed, I believe the `pprint` and `script_repr` functions won't work either. It looks like you might be able to use `pprint` on the `param` namespace instead...

Reply:
Thanks for catching that! Addressed in https://github.com/holoviz/param/pull/767/commits/7f606a7e618aa9e89d2736102f8e4a038f427d51 after adding some simple tests that would have caught this regression in https://github.com/holoviz/param/pull/778.